.github/workflows/test.yml

```diff
@@ -0,0 +1,69 @@
+name: tests
+
+on:
+  pull_request:
+    branches:
+      - "14.0*"
+  push:
+    branches:
+      - "14.0"
+      - "14.0-ocabot-*"
+
+jobs:
+  unreleased-deps:
+    runs-on: ubuntu-latest
+    name: Detect unreleased dependencies
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          for reqfile in requirements.txt test-requirements.txt ; do
+              if [ -f ${reqfile} ] ; then
+                  result=0
+                  # reject non-comment lines that contain a / (i.e. URLs, relative paths)
+                  grep "^[^#].*/" ${reqfile} || result=$?
+                  if [ $result -eq 0 ] ; then
+                      echo "Unreleased dependencies found in ${reqfile}."
+                      exit 1
+                  fi
+              fi
+          done
+  test:
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    name: ${{ matrix.name }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
+            makepot: "true"
+            name: test with Odoo
+          - container: ghcr.io/oca/oca-ci/py3.6-ocb14.0:latest
+            name: test with OCB
+    services:
+      postgres:
+        image: postgres:9.6
```

Comment:
According to docs Odoo 14.0 needs Postgres 10.0 or higher.

Reply:
Thanks for the thorough review. You have an good point here. 
https://www.odoo.com/documentation/14.0/administration/install/install.html#postgresql
>supported version: 10.0 and later

Just checked this with the actions config of 14.0-timesheet and 14.0-project repo.
They all run on postgres 9.6

This is something that should be changed in the repo template.
You might want to open a ticket there.